### PR TITLE
fix LightClient

### PR DESCRIPTION
**Pull request by syuhei176**
fixing 2 stuffs

* fix out of gas
* fix getBalance

### CODE REVIEW

File: packages/plasma-light-client/src/LightClient.ts [hunk 1]
```diff
@@ -198,7 +198,10 @@ export default class LightClient {
     const resultPromise = addrs.map(async addr => {
       const data = await this.stateManager.getVerifiedStateUpdates(
         Address.from(addr),
-        new Range(BigNumber.from(0), BigNumber.from(10000)) // TODO: get all stateUpdate method
+        new Range(
+          BigNumber.from(0),
+          BigNumber.from(JSBI.BigInt('10000000000000000000000'))
```
Comment (tkmct) [inline packages/plasma-light-client/src/LightClient.ts:203]: what about defining `MAX_BIG_NUMBER = JSBI.subtract(JSBI.exponentiate(JSBI.BigInt(2), JSBI.BigInt(256)), JSBI.BigInt(1))`

Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts:203]: maybe it's better to define in BigNumber class as static `BigNumber.MAX`

Comment (syuhei176) [reply packages/plasma-light-client/src/LightClient.ts:203]: OK, I'll try.

Comment (syuhei176) [reply packages/plasma-light-client/src/LightClient.ts]: fixed

**Pull request merged**